app.rb

```diff
@@ -0,0 +1,113 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+
+class App
+  def initialize(menu)
+    @menu = menu
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'No book found!'
+    else
+      @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    end
+    @menu.show_menu
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'No person found!'
+    else
+      @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+    end
+    @menu.show_menu
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase == 'y'
+    print 'Classroom: '
+    classroom = gets.chomp
+    @people.push(Student.new(classroom, age, name: name, parent_permission: parent_permission))
+    puts
+    puts 'Person created successfuly'
+  end
+
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name'
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    @people.push(Teacher.new(specialization, age, name: name))
+    puts
+    puts 'Person created successfuly'
+  end
```

Comment:
- Hey, great job, however, when trying to create a teacher, it throws an error of a wrong number of arguments, and this is basically how you've written the parameters in the constructor. See the image below to fix this issue:



![Screenshot from 2023-02-04 11-00-50](https://user-images.githubusercontent.com/59397159/216761130-25cd9b8b-4230-4cf9-93dd-8f0480b95079.png)



:point_right: *Please, kindly fix this error so that your app can work correctly after which you may submit for another review.*  :pray:

Reply:
### README FILE


- [ ] Hey, please, let's kindly update the contents and the issues page as seen in the image below to be in line with the current project as you've used the contents for another project. Kindly update this to make your readme more professional. See the images below:


![Screenshot from 2023-02-04 11-24-53](https://user-images.githubusercontent.com/59397159/216761909-1934eea0-fa3e-4bfd-a124-e4598705d7c0.png)
![Screenshot from 2023-02-04 11-24-06](https://user-images.githubusercontent.com/59397159/216761912-9eebb452-b1e2-415d-8a1a-5b81b2f0899b.png)